doc/getting-started/install.md

```diff
@@ -120,7 +120,7 @@ Download and install libsodium:
 ```bash
 git clone https://github.com/input-output-hk/libsodium
 cd libsodium
-git checkout 66f017f1
+git checkout a5baeccc
```

Comment:
We should use the latest commit used in iohk-nix 

https://github.com/input-output-hk/iohk-nix/blob/master/overlays/crypto/libsodium.nix#L10